pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -1224,17 +1224,18 @@ public byte[] GetAuditLogHash()
             using (var sha1 = new SHA1CryptoServiceProvider())
             {
                 var enc = Encoding.UTF8;
-                BlockHash blockHash = new BlockHash(sha1);
+                var blockHash = new BlockHash(sha1);
                 blockHash.ProcessBytes(enc.GetBytes(User));
                 if (!string.IsNullOrEmpty(ExtraInfo))
+                {
                     blockHash.ProcessBytes(enc.GetBytes(LogMessage.ParseLogString(ExtraInfo, LogLevel.all_info,
```

Comment:
I have to parse this string already to include expanded extra info element. Wouldn't it be better to use that value instead of parsing it again?

Reply:
I did this to avoid having to worry about hashing potentially modified english messages. Originally I was planning on not even reading the english strings from the XML since I don't use those in Skyline. I guess the resource name and its parameters could also be modified, but I think we should always prefer the data from the <message> and <name> elements.

When reading in the audit log and checking which log messages were modified I'll just construct a new audit log list and update the English strings for all entries that were modified. I also realized that some of my code that updates those English strings violates immutability, I'll fix that too.